src/redux/store.ts

```diff
@@ -1,9 +1,11 @@
 import { Action, configureStore, ThunkAction } from '@reduxjs/toolkit'
 import createTaskReducer from './features/createTaskSlice'
+import taskBoardReducer from './features/taskBoardSlice'
```

Comment:
Please prefer aliased imports

Reply:
I think we should setup prettier config for aliased import in this app. Better to do this in a separate PR I guess.

Reply:
Yes... it requires an extension and some hit-and-trial. I have it in my VSCode settings - will move it to prettierrc in a later PR :raised_hands: